tests/func/test_metrics.py

```diff
@@ -279,6 +279,42 @@ def test_formatted_output(self):
         assert expected_txt in stdout
         assert expected_json in stdout
 
+    def test_show_all_should_be_current_dir_agnostic(self):
+
+        metric_file = "mfile"
+
+        ret = main(
+            ["run", "-m", metric_file, "echo 0.5 >> {}".format(metric_file)]
+        )
+        self.assertEqual(0, ret)
+
+        os.chdir(self.DATA_DIR)
+
+        metrics = self.dvc.metrics.show(all_branches=True)
+
+        self.assertMetricsHaveRelativePaths(metrics)
```

Comment:
But `mfile` is not even in `metrics`.

Reply:
good point, at the beggining I didn't realize that this test sets up few metrics. There is no need for additional one. Also, there is no need to `run` any command, as we already have metrics on different branches.